pkg/app/piped/eventwatcher/eventwatcher.go

```diff
@@ -320,3 +324,51 @@ func convertStr(value interface{}) (out string, err error) {
 	}
 	return
 }
+
```

Comment:
nits, should we add a comment for this just as `modifyYAML` function has? It's a nits and it's up to you 🙆‍♀️